File: edu.Salflet/src/edu/Salflet/Servicios/MenuInterfaz.java
package edu.Salflet.Servicios;

import java.util.Scanner;

/**
 * contiene las cabeceras de los metodos a los que puede accederse desde el metodo main 
 * SAV - 05122023
 */
public interface MenuInterfaz {

	/**
	 * metodo encargado de mostrar nuestro menu un recoger la funcionalidad que el usuario desea
	 * SAV - 05122023
	 * @param sc
	 * @return
	 */
	public int menuSeleccion(Scanner sc);
}

File: salflet/Servicios/MenuInterfaz.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace salflet.Servicios
{
    /// <summary>
    /// contiene las cabeceras de los metodos a los que se puede llamar desde el metod Main
    /// SAV - 16012024
    /// </summary>
    internal interface MenuInterfaz
    {
        /// <summary>
        /// muestra las diferentes funciones de nuestra aplicacion, pide un tipo int y lo devuelve
        /// SAV - 16012024
        /// </summary>
        /// <returns></returns>
        public int menuSeleccion();
    }
}
